galaxy/settings/development.py

```diff
@@ -92,9 +92,3 @@
 ]
 
 STATIC_ROOT = ''
-
-# =========================================================
-# Metrics Settings
-# =========================================================
-
-METRICS_ENABLED = True
```

Comment:
I think we should keep `METRICS_ENABLED` setting and adopt it for new metrics implementation, so that InfluxDB service can be optional for development environment.

Reply:
Done. I renamed it to `GALAXY_METRICS_ENABLED`.